front/app/containers/IdeasEditPage/WithJSONForm/index.tsx

```diff
@@ -120,6 +158,33 @@ const IdeasEditPageWithJSONForm = ({ params: { ideaId } }: WithRouterProps) => {
     [uiSchema]
   );
 
+  const formRenderers = [
+    { tester: linearScaleControlTester, renderer: LinearScaleControl },
```

Comment:
@EdwinKato So since this list of renderers is used for the idea form (new and editing), should I instead accept a "configType" in the Form component, and then abstract these lists of renderers into another file like you mentioned in our meeting?